CVLByExample/ERC20/contracts/broken/ERC20.sol

```diff
@@ -407,4 +407,13 @@ contract ERC20 is IERC20, IERC20Metadata {
         (bool success, ) = msg.sender.call{value: amount}("");
         require(success);
     }
+
+
+    function depositAmount(uint256 amount) external {
```

Comment:
this is not deposit, lets think of a different example that is more realistic, maybe 
depositTo(address to, uint256 amount)  payable and check the msg.value > amount and the rest goes to msg.sender

Reply:
The "overflow" rules want to demonstrate that using require_uint hides the overflow and it doesn't happen with the changes you suggest. I changed the name for these rules and added the rule depositTo  for other places it is used.